sbt-scalaxb/src/main/scala/sbtscalaxb/Plugin.scala

```diff
@@ -30,7 +30,7 @@ object Plugin extends sbt.Plugin {
     lazy val packageDir       = SettingKey[Boolean]("scalaxb-package-dir")
     lazy val generateRuntime  = SettingKey[Boolean]("scalaxb-generate-runtime")
     lazy val protocolFileName = SettingKey[String]("scalaxb-protocol-file-name")
-    lazy val protocolPackageName  = SettingKey[Option[String]]("scalaxb-protocol-package-name")
+    lazy val protocolPackageName  = SettingKey[String]("scalaxb-protocol-package-name")
```

Comment:
Making this change would break every sbt-scalaxb user's build, which I don't think is worth it since existing users would likely have a `Some("foo")` set already, and they don't gain anything from it.